### PR TITLE
Make instructions more clear

### PR DESCRIPTION
**Pull request by pmillspaugh**

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -188,12 +205,29 @@ export async function getGitHubProfile() {
...
+/**
+ * ========================================================================================================================
+ * 🎉 Well done! 🎉
+ *
+ * Thanks for going through our getting-started tour!
+ * We know there's a lot to wrap your head around, and we want to make learning and using Membrane _way_ simpler.
+ *
+ * On that note, we'd love your feedback. You can reach us:
+ * 1. via this form (powered by a Membrane HTTP endpoint): https://spare-346-sector-257-manner-983-bet.hook.membrane.io
+ * 2. via email: juan@membrane.io, justin@membrane.io, pete@membrane.io
```
Comment (just-be-dev) [inline index.ts]:
```suggestion
 * 2. via email: contact@membrane.io
```

Let's use the shared google group so it goes to all of us.

File: index.ts [hunk 2]
```diff
@@ -11,11 +11,14 @@
...
+ * You might also want to open our docs https://docs.membrane.io in a separate tab to supplement the tutorial.
  *
- * To get started, open the Membrane Explorer and Membrane Logs in VS Code by opening your command palette and typing:
+ * To start, make sure the Membrane Explorer (sidebar) and Membrane Logs (bottombar) are open in your IDE.
+ * Open your command palette (cmd+shift+P on Mac) and type:
  * 1. "Membrane: Focus on Membrane Explorer View"
- * 2. "Membrane Logs: Focus on Membrane Logs View"
+ * 2. "Membrane Logs: Focus on Membrane Logs View" (then click the |• getting-started| tab to view logs for this program)
  *
+ * We'd love to hear your feedback: shoot an email to pete@membrane.io with any questions/suggestions/etc.
```
Comment (pmillspaugh) [inline index.ts]:
```suggestion
 * We'd love to hear your feedback: shoot an email to contact@membrane.io with any questions/suggestions/etc.
```

File: index.ts [hunk 3]
```diff
@@ -59,42 +64,47 @@ export async function run() {
...
  * ========================================================================================================================
  * STEP 3 | Membrane `sms` and `email`
  * ========================================================================================================================
  */
 
-// Membrane comes with several built-in utils that your programs can install, like `sms` and `email`.
-// Your workspace has `sms` and `email` installed, and this program includes them as connections.
-// We'll cover how to add new connections later.
+// Membrane comes with a few built-in programs that your programs can use, like `sms` and `email`.
+// This `getting-started` program already includes `sms` and `email` as connections (similar to npm dependencies).
```
Comment (just-be-dev) [inline index.ts:86]: Instead of saying `similar to npm dependencies, what if we just linked to the docs on connections?

Comment (just-be-dev) [reply index.ts:86]: Never mind, I see you have it linked below. It may be worth hoisting that up?

**Review by just-be-dev — APPROVED** [review]

File: index.ts (hunk 3 above)
Comment (pmillspaugh) [reply index.ts:86]: I'm hoping people may pick up on the pattern of link-to-learn-more at the bottom of each step, but that's probably wishful thinking. Maybe I could also add links right-justified in each step header?

Comment (just-be-dev) [reply index.ts:86]: I think the link to learn more pattern is fine

**Pull request merged**